### ISSUE TITLE
footer designing 

### ISSUE THREAD

**Issue by Vaibhav8090** [Bhavesh38/V-Book-Project#3]
you have to design the footer part of our website with various information like
about us
contact us
support 
site map
social media handles
partners

**Comment by niladrix719** [comment]
can i work on this?

**Comment by Vaibhav8090** [comment]
YES YOU CAN WORK FIRSTLY JOIN THE DISCORD AND GIVE SELF ROLE TO V BOOK AND CONTACT ME ON DISCORD THROUGH V BOOK CHANNEL
